src/main/java/seedu/taassist/logic/commands/ClassCommand.java

```diff
@@ -30,6 +31,11 @@ public ClassCommand(ModuleClass targetClass) {
     @Override
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
+
+        if (!model.hasModuleClass(targetClass)) {
+            throw new CommandException(String.format(MESSAGE_CLASS_NOT_FOUND, targetClass));
+        }
```

Comment:
Note: the message format has changed in `mater`. After resolving the conflict this should be: 
```suggestion
        if (!model.hasModuleClass(targetClass)) {
            throw new CommandException(String.format(Messages.MESSAGE_MODULE_CLASS_DOES_NOT_EXIST,
                    model.getModuleClassList()));
        }
```